### PR TITLE
Add spinning cursor activity indicator

### PR DESCRIPTION
**Pull request by kyleparrott**
Shows that there is activity during some commands that may take longer to run.

### CODE REVIEW

File: kubos [hunk 1]
```diff
@@ -227,5 +238,32 @@ def get_current_target():
     else:
         return None
 
+
+class StatusSpinner(threading._Timer):
```
Comment (marshall) [inline kubos]: I don't think `threading._Timer` is a public API, did you mean to use `threading.Timer` instead? (https://docs.python.org/2/library/threading.html#timer-objects)

Comment (kyleparrott) [inline kubos]: I came across the threading._Timer API which is built into python 2.7 but doesn't seem to be mentioned in the official python docs, http://fossies.org/dox/Python-2.7.11/classthreading_1_1__Timer.html 

If it still needs to change, thats a pretty straight forward fix.

Comment (rplauche) [inline kubos]: It looks like `threading._Timer` is the class and `threading.Timer` is just a factory function. I think the current inheritance is good.

**Comment by rplauche** [comment]
I think this is good to merge.

**Pull request merged**